Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Ride.hs

```diff
@@ -578,8 +578,10 @@ data FParamsRentalDetails = FParamsRentalDetails
     distBasedFare :: Money,
     timeBasedFareWithCurrency :: PriceAPIEntity,
     distBasedFareWithCurrency :: PriceAPIEntity,
+    deadKmFareWithCurrency :: PriceAPIEntity,
     extraDistance :: Meters,
-    extraDuration :: Seconds
+    extraDuration :: Seconds,
+    deadKmFare :: Money
```

Comment:
No need to have this `deadKmFare`. Only `deadKmFareWithCurrency` is enough. As older fields are there only because of backward compatibility.